app/config/FrontendAppConfig.scala

```diff
@@ -67,7 +67,7 @@ class FrontendAppConfig @Inject()(val servicesConfig: ServicesConfig, val config
 
   //Sign out
   lazy val ggUrl: String = servicesConfig.getString(s"government-gateway.url")
-  lazy val ggSignOutUrl = s"$ggUrl/gg/sign-out?continue=$exitSurveyUrl"
+  lazy val ggSignOutUrl = s"$ggUrl/bas-gateway/sign-out-without-state?continue=$exitSurveyUrl"
```

Comment:
Could you change comment on line 60 appropiately? Also remove line 62.

Reply:
resolved